deps/hiredis/async.c

```diff
@@ -62,18 +62,6 @@ static unsigned int callbackHash(const void *key) {
                                hi_sdslen((const hisds)key));
 }
 
-static void *callbackValDup(void *privdata, const void *src) {
```

Comment:
Let's not touch hiredis. We are looking to devendor it anyways, so I think they can make independent decisions about this.

Reply:
I think we must. 
i.e. I changed the type and then changed whatever did not compile.
will recheck if we must and remove if not

Reply:
You shouldn't need to change hiredis. Hiredis is compiled independently as a static file and then linked against.

Reply:
done